manifests/rhoai/shared/apps/model-catalog/model-catalog-unmanaged-sources.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: model-catalog-unmanaged-sources
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/managed: 'false'
+data:
+  modelCatalogSources: |
+    {
+      "sources": [
+          ]
+        }
```

Comment:
what's this? seems like duplicate `]}`?

Reply:
+1 - since we have no long contents we could even just make this value a single line string.

```yaml
data:
  modelCatalogSources: '{ "sources" : [] }'
```